src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -65,6 +65,7 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
     serviceLang,
     lang,
   } = useContext(ServiceContext);
+  const { enabled: preloadLeadImageToggle } = useToggle('preloadLeadImage');
```

Comment:
should we have a test for the `StoryPage` covering the use of the toggle and it's application to images?